sensor_msgs/Image[] images
---
Descriptor[] descriptors